libs/langchain/langchain/chains/base.py

```diff
@@ -289,39 +353,21 @@ def __call__(
             A dict of named outputs. Should contain all outputs specified in
                 `Chain.output_keys`.
         """
-        inputs = self.prep_inputs(inputs)
-        callback_manager = CallbackManager.configure(
-            callbacks,
-            self.callbacks,
-            self.verbose,
-            tags,
-            self.tags,
-            metadata,
-            self.metadata,
-        )
-        new_arg_supported = inspect.signature(self._call).parameters.get("run_manager")
-        run_manager = callback_manager.on_chain_start(
-            dumpd(self),
+        config = {
```

Comment:
why are we changing this? are we sure this doesnt have any side effects? eg with callbacks

Reply:
so that we can deprecate it. couldn't deprecate before bc invoke used it, now we flipped dependency

shouldn't change anything unless i made a mistake, can test some more